arbor/cable_cell_param.cpp

```diff
@@ -64,4 +69,74 @@ cable_cell_local_parameter_set neuron_parameter_defaults = {
     0.01
 };
 
+// Discretization policy implementations:
+
+locset cv_policy_max_extent::cv_boundary_points(const cable_cell& cell) const {
+    const auto& emorph = *cell.morphology();
+    const unsigned nbranch = emorph.morph().num_branches();
+    if (!nbranch || max_extent_<=0) return ls::nil();
+
+    std::vector<mlocation> points;
+
+    unsigned bidx = 0;
+    if (flags_&cv_policy_flag::single_root_cv) {
```

Comment:
Would this generate a correct discretisation if `single_root_cv` was set without a spherical root?

Reply:
The issue of whether or not to deal specially with a spherical root CV at discretization has to be pushed down to the actual discretizer, as an explicit discretization, for example, may include points on the root branch that are interior also.

So the locsets produced by CV policies should be agnostic when it comes to spherical or cable root branches, and the decision regarding indivisible spherical somata deferred to later.